### PR TITLE
Navbar component

### PR DESCRIPTION
**Pull request by arjun-computer-geek**
create a navbar component.

### CODE REVIEW

File: components/badge.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
+.badge-container{
+    position: relative;
+    width: fit-content;
+    box-sizing: border-box;
+}
+.badge{
+    padding: 0.2rem 0.65rem;
+    color: var(--white);
```
Comment (Shailseen) [inline components/badge.css:8]: var name instead of --white may be --white-color.

**Review by sagargse — APPROVED** [review]
looks good you can go ahead

File: components/alert.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
+.alert {
+    border: 2px solid;
```
Comment (Shailseen) [inline components/alert.css:2]: can you give a different class for using border more than one times.

File: components/button.css [hunk 3]
```diff
@@ -0,0 +1,130 @@
+.btn {
+    padding: 1rem;
+    border: 2px solid;
```
Comment (Shailseen) [inline components/button.css:3]: here also change for the border as the mention above.

**Review by Shailseen — APPROVED** [review]
need some minor changes. all looking good for me.

**Pull request closed without merging**